src/SharedFrameworkValidation/ApiCompatBaseline.LKGRuntime.CurrentRuntime.txt

```diff
@@ -0,0 +1,875 @@
+Compat issues with assembly Microsoft.CodeAnalysis:
```

Comment:
Sure, I just thought that we actually wanted to keep track of all of the differences in this baseline, but yeah, I'll send out a new PR that excludes Roslyn.